### PR TITLE
Enable JWT expiration by default

### PR DESCRIPTION
**Pull request by maarcingebala**
Enable JWT expiration by default.

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

File: saleor/settings.py [hunk 1]
```diff
@@ -674,8 +674,7 @@ def SENTRY_INIT(dsn: str, sentry_opts: dict):
 CACHES = {"default": django_cache_url.config()}
 CACHES["default"]["TIMEOUT"] = parse(os.environ.get("CACHE_TIMEOUT", "7 days"))
 
-# Default False because storefront and dashboard don't support expiration of token
-JWT_EXPIRE = get_bool_from_env("JWT_EXPIRE", False)
+JWT_EXPIRE = True
```
Comment (korycins) [inline saleor/settings.py:677]: Shouldn't we still have `get_bool_from_env` here but with the default value set to `True`?

Comment (maarcingebala) [reply saleor/settings.py:677]: I think it should be always enabled. If you want to disable it locally, use `local_settings.py`.

**Review by L3str4nge — APPROVED** [review]

File: saleor/settings.py (hunk 1 above)
Comment (IKarbowiak) [reply saleor/settings.py:677]: I agree with @korycins, the environment variables aren't exactly for such a case like this when you want to override existing settings?

Comment (maarcingebala) [reply saleor/settings.py:677]: Having it disabled is a security issue so I'm not sure it should be even configurable.

**Review by IKarbowiak — APPROVED** [review]

**Comment by maarcingebala** [comment]
Docs: https://github.com/saleor/saleor-docs/pull/401

**Pull request merged**